msal_extensions/windows.py

```diff
@@ -120,4 +122,4 @@ def unprotect(self, cipher_text):
             finally:
                 _LOCAL_FREE(result.pbData)
         err_code = _GET_LAST_ERROR()
-        raise OSError(None, _err_description.get(err_code), None, err_code)
+        raise OSError(None, _err_description.get(err_code, ''), None, err_code)
```

Comment:
We should use an empty string `''` as the default value. Otherwise, it becomes:

```
[Errno 0] Decryption failed: [WinError 0] None App developer may consider this guidance: https://github.com/AzureAD/microsoft-authentication-extensions-for-python/wiki/PersistenceDecryptionError: 'C:\\Users\\xxx\\.azure\\msal_token_cache.bin'
```